layers/sqlsrv/Dockerfile

```diff
@@ -17,14 +17,17 @@ RUN pecl install -f pdo_sqlsrv
 RUN cp /opt/bref/lib/php/extensions/no-debug-non-zts-*/sqlsrv.so /tmp/sqlsrv.so
 RUN cp /opt/bref/lib/php/extensions/no-debug-non-zts-*/pdo_sqlsrv.so /tmp/pdo_sqlsrv.so
 
+RUN echo $'extension=/opt/bref-extra/sqlsrv.so\n\
+extension=/opt/bref-extra/pdo_sqlsrv.so' > /tmp/ext.ini
```

Comment:
Is there a problem to have both enabled?

Reply:
Should be fine. I have had them both enabled in the past with no issues.

Reply:
Thank you